wrappers/android/examples/devices_presets/src/main/res/values-night/themes.xml

```diff
@@ -0,0 +1,16 @@
+<resources xmlns:tools="http://schemas.android.com/tools">
```

Comment:
Please remove all the png files that have been uploaded, or explain why they are needed